common/fw/CMakeLists.txt

```diff
@@ -30,11 +30,17 @@ message(STATUS "T26X_FW_VERSION: ${T26X_FW_VERSION}")
 set(T26X_FW_SHA1 c3940ccbb0e3045603e4aceaa2d73427f96e24bc)
 set(T26X_FW_URL "${REALSENSE_FIRMWARE_URL}/Releases/TM2/FW/target/${T26X_FW_VERSION}")
 
-string(REGEX MATCH "L5XX_RECOMMENDED_FIRMWARE_VERSION \"([0-9]+.[0-9]+.[0-9]+.[0-9]+)\"" _ ${ver})
-set(L5XX_FW_VERSION ${CMAKE_MATCH_1})
-message(STATUS "L5XX_FW_VERSION: ${L5XX_FW_VERSION}")
-set(L5XX_FW_SHA1 ab73e5bfc520c0aa0340cada4b3e317b8fd31a4d)
-set(L5XX_FW_URL "${REALSENSE_FIRMWARE_URL}/Releases/L5xx/FW")
+string(REGEX MATCH "L51X_RECOMMENDED_FIRMWARE_VERSION \"([0-9]+.[0-9]+.[0-9]+.[0-9]+)\"" _ ${ver})
```

Comment:
Shouldn't the `.` be `\.`?

Reply:
you mean between the digits?  Its like all the rests. and it works...
string(REGEX MATCH "D4XX_RECOMMENDED_FIRMWARE_VERSION \"([0-9]+.[0-9]+.[0-9]+.[0-9]+)\"" _ ${ver})

Reply:
I'm sure it works, but it will catch `1x2x3x4`, too...

Reply:
done for L51X and L53X